server/app/views/NorthStarBaseView.java

```diff
@@ -58,6 +58,7 @@ protected ThymeleafModule.PlayThymeleafContext createThymeleafContext(
     ThymeleafModule.PlayThymeleafContext context = playThymeleafContextFactory.create(request);
     context.setVariable("tailwindStylesheet", assetsFinder.path("stylesheets/tailwind.css"));
     context.setVariable("uswdsStylesheet", assetsFinder.path("dist/uswds.min.css"));
+    context.setVariable("northStarStylesheet", assetsFinder.path("dist/north-star-styles.css"));
```

Comment:
As mentioned above, use `dist/northstar-styles.min.css`. I renamed it because I kept typing it wrong